chromium/browser_type.go

```diff
@@ -164,6 +165,9 @@ func (b *BrowserType) flags(lopts *common.LaunchOptions) map[string]interface{}
 		f["mute-audio"] = true
 		f["blink-settings"] = "primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4"
 	}
+
+	setFlagsFromArgs(f, lopts.Args)
+
```

Comment:
No, sorry, don't think that's a good idea. `flags()` has a single purpose: setting flags from `LaunchOptions` (and now `k6lib.Options`). Since reading `l.Args` is more complex and involves messy string parsing, that's an implementation detail as far as `flags()` is concerned. I definitely don't want to see that while I'm reading `flags()`, so a separate function makes more sense to me.

Reply:
Yes, it's an implementation detail and that's why I suggested putting it in the implementation where it's only needed :) We can still put this inside the same function as a closure if what you want is single purpose code :) I don't think we need reusability here for another function.

Reply:
No, I mean that it's an implementation detail for `flags()`. It's not about using a closure or not; it's about different responsibilities and levels of abstraction. If I'm reading `flags()` I don't want to see string parsing(!), regardless if it's inline or in an inline closure. It just doesn't belong there.

The separation also makes the code more unit testable, since we could easily write isolated unit tests for `setFlagsFromArgs()`. I just chose to keep the "unit" at the `flags()` level, since we already had tests for it.

Reply:
Alright, then.